src/ImageSharp/Common/Helpers/Vector512Utilities.cs

```diff
@@ -126,6 +128,13 @@ public static Vector512<float> RoundToNearestInteger(Vector512<float> vector)
             return Avx512F.RoundScale(vector, 0b0000_1000);
         }
 
+        if (Avx.IsSupported)
+        {
+            Vector256<float> lower = Avx.RoundToNearestInteger(vector.GetLower());
+            Vector256<float> upper = Avx.RoundToNearestInteger(vector.GetUpper());
+            return Vector512.Create(lower, upper);
+        }
```

Comment:
What's the AVX path for with Vector512?

`Vector512.IsHardwareAccelerated` will only report true if `Avx512F+BW+CD+DQ+VL` is supported, so this path should generally be "dead".